hack/feature-delete.sh

```diff
@@ -0,0 +1,36 @@
+#!/bin/bash
+
+set -e
+
+if [ "$FEATURES_ENVIRONMENT" == "" ]; then
+	echo "[ERROR]: No FEATURES_ENVIRONMENT provided"
+	exit 1
+fi
+
+if [ "$FEATURES" == "" ]; then
+	echo "[ERROR]: No FEATURES provided"
+	exit 1
+fi
+
+# expect oc to be in PATH by default
+export OC_TOOL="${OC_TOOL:-oc}"
+
+# Destroy features
+
+for feature in $FEATURES; do
+
+  feature_dir=feature-configs/${FEATURES_ENVIRONMENT}/${feature}/
+  if [[ ! -d $feature_dir ]]; then
+    echo "[WARN] Feature '$feature' is not configured for environment '$FEATURES_ENVIRONMENT', skipping it"
+    continue
+  fi
+
+  echo "[INFO] Deleting feature '$feature' for environment '$FEATURES_ENVIRONMENT'"
+    set +e
+    if ! ${OC_TOOL} delete -k $feature_dir
```

Comment:
Can you put the `then` on the same line to keep consistency with above?

Reply:
Fixed, but this snuck in from the original file (I just copied `feature-deploy.sh` and removed a bunch of logic)

https://github.com/openshift-kni/cnf-features-deploy/blob/master/hack/feature-deploy.sh#L39

Would it be worth touching the deploy file with this PR?

Reply:
I am not against it